src/SmartEnum/Core/DoNotExecute.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Ardalis.SmartEnum.Core
```

Comment:
The namespace can be simply `Ardalis.SmartEnum`.  This saves the users to have to add `using` clauses or `Core` for all interface uses.

Reply:
I put these classes into a "Core" namespace as they're not intended to be used by users. So when a user does `using Ardalis.SmartEnum` they don't get these somewhat cryptic implementation details in intellisense.

I'm happy to move them to `Ardalis.SmartEnum` if that's where you'd prefer them.

Reply:
I didn't think of that. That's a good idea to keep `Core`.